Billing/billing-process-guidelines.md

```diff
@@ -0,0 +1,33 @@
+{{{
+  "title": "Ecosystem Partner Billing Setup Process",
+  "date": "02-17-2016",
+  "author": "Andrew Brunette",
+  "attachments": [],
+  "contentIsHTML": false,
+  "sticky": true
+}}}
+
+## Ecosystem Partner Billing Setup Process
+
+### Description
+This document describes the process and responsible parties for setting an Ecosystem Partner product up to be billed to the customer when the Blueprint that loads the product is run.
+
+### Audience
+Ecosystem and sales team members who are working with Ecosystem partner companies that wish to bill for their products in the Ecosystem Marketplace.
+
+| Step Name | Description | Responsible Team(s) |
+| --- | --- | --- |
+| Provider Signup | Provider signs up as a customer, and agrees to Supplemental Terms which establishes them legally as a Provider Partner. | Ecosystem |
+| Provider Integration | Provider completes an integration | Ecosystem |
+| Billing Model Definition | Provider Partner and ecosystem team member define the product and pricing model functionally. Select products, configurations, define price level and billing trigger for each item.  This is a design task, defining the billing model to be implemented in subsequent steps. This pricing is defined at net levels to provider | Ecosystem |
+| SKU Setup | Setting up the SKU in Platform.  This will be done by creating a ticket to Mary Cadera/Jared Ruckle (need to define these people functionally, rather than by name).  Cadera/Ruckle will create the SKU so the product can be billed. | Ecosystem to create ticket, Platform to instantiate SKUs. |
```

Comment:
is this really ready?  why are we naming people in a kb specifically ?  seems like this isn't complete